src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -67,17 +58,12 @@ public CaseSearchResult searchCases(
             + "the search results will contain only metadata by default (no case field data). To get case data in the "
             + "search results, please state the alias fields to be returned in the _source property for e.g."
             + " \"_source\":[\"alias.customer\",\"alias.postcode\"]",
-                  required = true)
+            required = true)
         @RequestBody String jsonSearchRequest) {
 
         Instant start = Instant.now();
 
-        rejectBlackListedQuery(jsonSearchRequest);
-
-        CrossCaseTypeSearchRequest request = new CrossCaseTypeSearchRequest.Builder()
-            .withCaseTypes(caseTypeIds)
-            .withSearchRequest(stringToJsonNode(jsonSearchRequest))
-            .build();
+        CrossCaseTypeSearchRequest request = elasticsearchQueryHelper.prepareRequest(caseTypeIds, jsonSearchRequest);
```

Comment:
I like the idea of having common code in ElasticsearchQueryHelper but because we are moving everything in there the controllers are almost empty and ElasticsearchQueryHelper is quite complex. We could improve things by moving the code in `prepareRequest` back to the controllers. And leave the utility methods common to both in ElasticsearchQueryHelper. I prefer to see in the controller method itself the steps it's taking.